Bottom: 16d59423817af4597d3400925cc85c8ff80fde8c
Top:    972d95f14bea58046380afbadee2106aac8095aa
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-24 01:43:18 +0000

find.c: Tidy up some macro definitions

(Whitespace only changes)


---

diff --git a/src/find.c b/src/find.c
index 34247a1..3d1e596 100644
--- a/src/find.c
+++ b/src/find.c
@@ -119,26 +119,13 @@
 		fputc('\n', (FP));						\
 	}
 
-#define	LIST_ENTRY(list,I)	\
-	(((AnyObjectType **)list->Data)[(I)])
-
-#define	PADLIST_ENTRY(L,I)	\
-	(((PadType **)PadList[(L)].Data)[(I)])
-
-#define	LINELIST_ENTRY(L,I)	\
-	(((LineType **)LineList[(L)].Data)[(I)])
-
-#define	ARCLIST_ENTRY(L,I)	\
-	(((ArcType **)ArcList[(L)].Data)[(I)])
-
-#define RATLIST_ENTRY(I)	\
-	(((RatType **)RatList.Data)[(I)])
-
-#define	POLYGONLIST_ENTRY(L,I)	\
-	(((PolygonType **)PolygonList[(L)].Data)[(I)])
-
-#define	PVLIST_ENTRY(I)	\
-	(((PinType **)PVList.Data)[(I)])
+#define LIST_ENTRY(list,I)      (((AnyObjectType **)list->Data)[(I)])
+#define PADLIST_ENTRY(L,I)      (((PadType **)PadList[(L)].Data)[(I)])
+#define LINELIST_ENTRY(L,I)     (((LineType **)LineList[(L)].Data)[(I)])
+#define ARCLIST_ENTRY(L,I)      (((ArcType **)ArcList[(L)].Data)[(I)])
+#define RATLIST_ENTRY(I)        (((RatType **)RatList.Data)[(I)])
+#define POLYGONLIST_ENTRY(L,I)  (((PolygonType **)PolygonList[(L)].Data)[(I)])
+#define PVLIST_ENTRY(I)         (((PinType **)PVList.Data)[(I)])
 
 #define IS_PV_ON_RAT(PV, Rat) \
 	(IsPointOnLineEnd((PV)->X,(PV)->Y, (Rat)))
